api/src/env.ts

```diff
@@ -45,6 +45,15 @@ const allowedEnvironmentVars = [
 	'REFRESH_TOKEN_COOKIE_SECURE',
 	'REFRESH_TOKEN_COOKIE_SAME_SITE',
 	'REFRESH_TOKEN_COOKIE_NAME',
+
+	'REDIS_NAMESPACE',
+	'REDIS_CONNECTION_STRING',
```

Comment:
Should it be `REDIS` instead of `REDIS_CONNECTION_STRING`?

```suggestion
	'REDIS',
	'REDIS_NAMESPACE',
```